include/utils.h

```diff
@@ -168,4 +159,140 @@ inline std::vector<Perf> createVectorPerfWithNoPerf(std::string id,
   return vp;
 }
 
+/**
+ * Extract only performance values from a vector of Perf objects
+ * @param p vector of Perf objects
+ *
+ */
+inline std::vector<double> getPerfFromPerfVect(std::vector<Perf> &p) {
+  std::vector<double> values;
+  for (auto perf : p) {
+    values.push_back(static_cast<double>(perf.getValue()));
+  }
+  return values;
+}
+
+/**
+ * Check if a key is present in a unordered map
+ * @param map unordered map of <int,int>
+ *
+ */
+inline bool checkKey(std::unordered_map<int, int> &map, int key) {
```

Comment:
I'm wondering why adding a function that does just this line `map.find(key) != map.end()` ?